src/Fabulous.Avalonia/Widgets/Media/Brushes/LinearGradientBrush.fs

```diff
@@ -36,6 +43,14 @@ module LinearGradientBrushBuilders =
                 LinearGradientBrush.EndPoint.WithValue(RelativePoint(endPoint, unit))
             )
 
+        static member inline LinearGradientBrush<'msg>(startPoint: Point, endPoint: Point) =
+            CollectionBuilder<'msg, IFabLinearGradientBrush, IFabGradientStop>(
+                LinearGradientBrush.WidgetKey,
+                GradientBrush.GradientStops,
+                LinearGradientBrush.StartPoint.WithValue(RelativePoint(startPoint, RelativeUnit.Relative)),
+                LinearGradientBrush.EndPoint.WithValue(RelativePoint(endPoint, RelativeUnit.Relative))
+            )
+
```

Comment:
When adding extra constructors that are just converting arguments to another type, you could do like ImageSource to avoid repeating the same code:

```suggestion
        static member inline LinearGradientBrush<'msg>(startPoint: Point, endPoint: Point) =
            let startPoint = RelativePoint(startPoint, RelativeUnit.Relative)
            let endPoint = RelativePoint(endPoint, RelativeUnit.Relative)
            View.LinearGradientBrush<'msg>(startPoint, endPoint)

```